### ISSUE TITLE
Admin dashboard drag and drop in any cell position

### ISSUE THREAD

**Issue by giannik** [OrchardCMS/OrchardCore#8769]
The new admin dashboard is great.
I was wondering if it would be possible when arranging widgets using drag and drop  to drop in any of the  container cells . so for example 1 widget in the far left and one in the far right , leaving an empty cell in the middle.
Or for example drop a widget in the bottom right corner  as shown below :

![image](https://user-images.githubusercontent.com/1516981/110207377-d4c27600-7e8b-11eb-897e-ce63f00c30c2.png)

@ns8482e

**Comment by sebastienros** [comment]
It's not really a grid because it's responsive. I don't know if it can go from 6 to 3 columns for instance in an intermediate screen size. I know for sure that it will go to 1 column on small screens, in which case it's fine to convert.

Even if the layout could be replaced, the current object model would not allow for that. A solution might be to add fake empty widgets. So doing would mean reimplement the way it's currently done.

**Comment by giannik** [comment]
Yes sebastien , thought of that too. Add dummy div elements by the number of cols*rows. to act as containers that will be placeholders to drag elements into.
My idea was to add  a footer widget at the bottom to span the entire last row , regardless of the top widgets. 
Using css from within chrome dev i can acheive it by setting the css properties : grid-column and grid-row on the widget  , and and then these would need to be persisted. But could not get it working from the UI , with drag and drop.

**Comment by ns8482e** [comment]
yes, Technically it's possible, However current object model use `Position` (int) that stores the Sortable index.
